### ISSUE TITLE
passenger restart using touch tmp/restart.txt when passenger-config is available

### ISSUE THREAD

**Issue by JeanMertz** [capistrano/passenger#11]
The restart command checks if the version is greater than 4 (so it needs `5`, not `4.x`). We are running `4.0.59` and this command works perfectly for us:

``` bash
sudo passenger-config restart-app /var/www/staging --ignore-app-not-running
```

However, due to the precondition check, the `deploy:restart` still does a `touch tmp/restart.txt`, which doesn't seem to work in our case (that is, we don't see a restart of the Passenger instances, their started-at date stays at the old date).

**update** okay, after reading some more, I now understand that `touch tmp/restart.txt` actually does a restart of the Passenger instance on _the next incoming request_. So this is why we didn't see a restart before initialising a new request. It now restarts correctly. _However_, since `passenger-config` is present and working (and seems to be the better option of the two), I think there should be a way to force using that method if it's available.

**Comment by betesh** [comment]
I would accept a PR for this

**Comment by betesh** [comment]
Resolved by #12
